packages/react-storage/src/components/StorageBrowser/context/elements/__tests__/IconElement.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { IconElement, IconVariant, DEFAULT_ICON_PATHS } from '../IconElement';
+
+describe('IconElement', () => {
+  it('should render icon', () => {
+    const { container } = render(<IconElement />);
+
+    expect(container).toBeInTheDocument();
+  });
+
+  it('should apply default icon attributes', () => {
+    const { container } = render(<IconElement />);
+
+    const svgElement = container.querySelector('svg');
+
+    expect(svgElement).toHaveAttribute('aria-hidden', 'true');
+    expect(svgElement).toHaveAttribute('width', '24');
+    expect(svgElement).toHaveAttribute('height', '24');
+    expect(svgElement).toHaveAttribute('viewBox', '0 -960 960 960');
+    expect(svgElement).toHaveAttribute('fill', 'none');
+    expect(svgElement).toHaveAttribute('xmlns', 'http://www.w3.org/2000/svg');
+  });
+
+  it.each(Object.entries(DEFAULT_ICON_PATHS))(
+    'should render correct path for a %s `Icon` variant',
+    (variant, expectedPath) => {
+      const { container } = render(
+        <IconElement variant={variant as IconVariant} />
```

Comment:
Casting here because of typescript was complaining that `variant` is a string

Reply:
Don't think you need to update this PR but generally prefer flagging with `// @ts-expect-error - explanation`  to provide future maintainers with reasoning